sympy/physics/control/tests/test_lti.py

```diff
@@ -0,0 +1,258 @@
+                [-a0, -a1, -a2]]),
+        Matrix([0, 0, 1]),
+        Matrix([[b0, b1, b2]]),
+        zeros(1)
+    )
+    expect = StateSpaceModel(
+        Matrix([[0, 1, 0, 0, 0],
+                [-a0, -a1, 0, 0, 0],
+                [0, 0, 0, 1, 0],
+                [0, 0, 0, 0, 1],
+                [b0, b1, -a0, -a1, -a2]]),
+        Matrix([0, 1, 0, 0, 0]),
+        Matrix([[0, 0, b0, b1, b2]]),
+        zeros(1)
+    )
+    assert ssm1.series(ssm2).represent == expect.represent
+
+
+def test_TransferFunctionModel_series():
+    a, b, s = symbols('a, b, s')
+    tfm1 = TransferFunctionModel(
+        Matrix([1 / (a * s)])
+    )
+    tfm2 = TransferFunctionModel(
+        Matrix([(b + s) / (a + s)])
+    )
+    expect = TransferFunctionModel(
+        Matrix([(b + s) / (a * s * (a + s))])
+    )
+    assert tfm1.series(tfm2).G == expect.G
+
+    H1 = TransferFunctionModel(
+        Matrix([7 / (s**9 + 9*s)])
+    )
+    H2 = TransferFunctionModel(
+        Matrix([5 / (s + 2)])
+    )
+    H = TransferFunctionModel(
+        Matrix([35 / ((s + 2) * (s**9 + 9*s))])
+    )
+    assert H1.series(H2).G == H.G
+
+
+def test_StateSpaceModel_parallel():
+    a0, a1, a2, b0, b1, b2 = symbols('a:3, b:3')
+    ssm1 = StateSpaceModel(
+        Matrix([[0, 1],
+               [-a0, -a1]]),
+        Matrix([0, 1]),
+        Matrix([[b0, b1]]),
+        zeros(1)
+    )
+    ssm2 = StateSpaceModel(
+        Matrix([[0, 1, 0],
+                [0, 0, 1],
+                [-a0, -a1, -a2]]),
+        Matrix([0, 0, 1]),
+        Matrix([[b0, b1, b2]]),
+        zeros(1)
+    )
+    expect = StateSpaceModel(
+        Matrix([[0, 1, 0, 0, 0],
+                [-a0, -a1, 0, 0, 0],
+                [0, 0, 0, 1, 0],
+                [0, 0, 0, 0, 1],
+                [0, 0, -a0, -a1, -a2]]),
+        Matrix([0, 1, 0, 0, 1]),
+        Matrix([[b0, b1, b0, b1, b2]]),
+        zeros(1)
+    )
+    assert ssm1.parallel(ssm2).represent == expect.represent
+
+
+def test_TransferFunctionModel_parallel():
+    a, b, s = symbols('a, b, s')
+    tfm1 = TransferFunctionModel(
+        Matrix([1 / (a * s)])
+    )
+    tfm2 = TransferFunctionModel(
+        Matrix([(b + s) / (a + s)])
+    )
+    expect = TransferFunctionModel(
+        Matrix([(a + s + a * s * (b + s)) / (a * s * (a + s))])
+    )
+    assert together(tfm1.parallel(tfm2).G) == together(expect.G)
+
+    H1 = TransferFunctionModel(
+        Matrix([7 / (s**9 + 9*s)])
+    )
+    H2 = TransferFunctionModel(
+        Matrix([5 / (s + 2)])
+    )
+    H = TransferFunctionModel(
+        Matrix([7 / (s**9 + 9*s) + 5 / (s + 2)])
+    )
+    assert together(H1.parallel(H2).G) == together(H.G)
+
+
+def test_equality():
+    a0, a1, b0, b1 = symbols('a:2, b:2')
+    ssm1 = StateSpaceModel(
+        Matrix([[0, 1],
+               [-a0, -a1]]),
+        Matrix([0, 1]),
+        Matrix([[b0, b1]]),
+        zeros(1)
+    )
+    tfm = TransferFunctionModel(ssm1)
+    ssm2 = StateSpaceModel(tfm)
```

Comment:
What is the reasoning behind have "Model" in the class names? It seems good for state space, i.e. a state space model or state space system, but it seems unnecessary for a transfer function. Are you adding "Model" to the transfer function class just to copy the state space one?

Reply:
We can remove "Model" from both class names. So it would be `StateSpace` and `TransferFunction` - just like we have in python-control.

Reply:
If that is what Python Control has, it seems good to have the same names. "Model" doesn't add much besides more typing.